src/core/Str.pm

```diff
@@ -265,6 +282,7 @@ my class Str does Stringy { # declared in BOOTSTRAP
           nqp::p6box_i($i)
         )
     }
+    multi method rindex(Str:D: Cool:D $needle, Num:D $pos) {self.rindex: $needle.Str, $pos.Int}
```

Comment:
Not sure where this candidate came from. It did not exist before this PR. Did you mean `Cool:D` instead?